spikewrap/process/_preprocessing.py

```diff
@@ -120,6 +119,12 @@ def _get_pp_funcs() -> dict[str, Callable]:
         "phase_shift": si.phase_shift,
         "bandpass_filter": si.bandpass_filter,
         "common_reference": si.common_reference,
+        "whiten": si.whiten,  # Add whiten function
     }
 
     return pp_funcs
+
+
+def _preprocess_recording(recording, params=None):
```

Comment:
I think this was needed due to the change of function name above. This function should be preprocessing the passed dictionary but has disappeared with the function rename.